python/lsst/cp/pipe/ptc.py

```diff
@@ -404,12 +404,18 @@ def runDataRef(self, dataRef, visitPairs):
             expTime = exp1.getInfo().getVisitInfo().getExposureTime()
 
             tupleRows = []
+            nAmpsNan = 0
             for ampNumber, amp in enumerate(detector):
                 ampName = amp.getName()
                 # covAstier: (i, j, var (cov[0,0]), cov, npix)
                 doRealSpace = self.config.covAstierRealSpace
                 muDiff, varDiff, covAstier = self.measureMeanVarCov(exp1, exp2, region=amp.getBBox(),
                                                                     covAstierRealSpace=doRealSpace)
+                if np.isnan(muDiff) and np.isnan(varDiff) and np.isnan(covAstier):
```

Comment:
Are you OK with some but not all of these being NaN? Or do you want `or`s here?

Reply:
I'll put `or`s to report any NaN in the covariances too.